openlibrary/core/edits.py

```diff
@@ -72,20 +71,25 @@ def get_requests(cls, limit: int = 50, page: int = 1, mode: str = 'all', order:
 
         data = {}
         status_wheres = []
-
-        for i, status in enumerate(statuses):
-            data[f'status_{i}'] = status
-            status_wheres.append(f'status=$status_{i}')
+        if mode != 'all':  # No need to add every status to the WHERE clause
+            for i, status in enumerate(statuses):
+                data[f'status_{i}'] = status
+                status_wheres.append(f'status=$status_{i}')
```

Comment:
```suggestion
            data = {f'status_{i}': status for i, status in enumerate(statuses)}
        status_wheres = [f'status=${status}' for status in data]
```
and the definition of `status_wheres` on line 73 can be deleted.